parsers/jscript.c

```diff
@@ -2352,7 +2369,16 @@ static bool parseMethods (tokenInfo *const token, int class_index,
 						else if (is_setter)
 							kind = JSTAG_SETTER;
 
-						index_for_name = makeJsTag (name, kind, signature, NULL);
+						if (is_private)
+						{
+							vString * s = vStringNew ();
+							vStringPut (s, '#');
```

Comment:
You can unify the two function calls into one with:
```
vString * s = vStringNewInit ("#");
```

I don't say this is efficient:-).